bridge/server/lib/services/DatastoreService.js

```diff
@@ -35,6 +35,26 @@ class DatastoreService {
       mappedEvent.eventTypeHeadline = 'Remediation';
     }
 
+    //mappedEvent.data.project = "test";
+
+    if (event.source === 'dynatrace') {
+      var dttags = event.data.Tags.split(', ');
+      console.log(dttags);
+      var i;
+      for (i = 0; i < dttags.length; i++) {
+        if (dttags[i].indexOf("service:") == 0) {
```

Comment:
Does this whole blocks till work with the new tags we are using?
![Screenshot from 2019-12-10 12-35-17](https://user-images.githubusercontent.com/56065213/70701147-2ea9af80-1ccc-11ea-8637-cdb2674bef6e.png)

Reply:
This is no longer needed as it is handled by dynatrace-service.